test/check-application

```diff
@@ -155,6 +155,7 @@ class TestApplication(testlib.MachineCase):
         self.has_criu = "debian" not in m.image and "ubuntu" not in m.image
         self.has_selinux = not any(img in m.image for img in ["arch", "debian", "ubuntu", "suse"])
         self.has_cgroupsV2 = not m.image.startswith('rhel-8')
+        self.supports_quadlet = not any(img in m.image for img in ["ubuntu-2204"])
```

Comment:
Ah sorry, [debian stable has 4.3](https://packages.debian.org/bookworm/podman), so that needs to be excluded, too. But TBH I have troubles parsing this "three levels nested" loop. I think what it does is to compare each *letter* in "ubuntu-2204" against m.image, which really isn't what you want :grin: 

Could you just do

    supports_quadlet = img not in ["debian-stable", "ubuntu-2204"]

please?